bundles/pcmumlclass/tools.vitruv.applications.pcmumlclass/src/tools/vitruv/applications/pcmumlclass/uml/UmlInterface.reactions

```diff
@@ -41,18 +41,16 @@ routine insertCorrespondingInterface(uml::Interface umlInterface, uml::Package u
 routine detectOrCreateCorrespondingInterface(uml::Interface umlInterface, uml::Package umlPackage) {
```

Comment:
This routine already implements the find-or-create-pattern, so no larger changes are required here.